embulk-input-s3/src/main/java/org/embulk/input/s3/AlwaysRetryable.java

```diff
@@ -155,4 +179,29 @@ public T executeWith(RetryExecutor executor)
             throw Throwables.propagate(e);
         }
     }
+
+    private static boolean isAse(SdkBaseException e)
```

Comment:
I think an inline `e instanceof AmazonServiceException` is much clearer, this is kind of a cryptic method name. Inside the aws-sdk's implementation, they use this name because they have a family of isXXX for checking the error type, consistency counts in their case. For us, it just doesn't worth it.